src/common/cis2pi.jl

```diff
@@ -1,5 +1,15 @@
 """Function to compute exp(2π i x)"""
 cis2pi(x) = cispi(2x)
```

Comment:
Is this needed at all. I mean could this not just be folded into the definition below ?

Reply:
Yes, the special casing is incorrect for ForwardDiff (it will silently ignore the dual part)